docs/_future/manifest_workflows.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Manifest workflows
+order: 4
+slug: manifest-workflows
+lang: en
+layout: future
+---
+
+<div class="section-content">
+
+Your app can use Functions by referencing them in Workflows. Your Functions and the ones provided by Slack can be used as steps in Workflow definitions.
```

Comment:
I think the second sentence here is making references to both custom functions (i.e. "Your functions") as well as built-in functions (i.e. "and the ones provided by Slack"). As such, let's use the same terms we use on the api.slack.com site: Custom Functions and Built-in Functions - and let's also link to the relevant documentation:

- Custom: https://api.slack.com/future/functions/custom
- Built-in: https://api.slack.com/future/functions